scripts/changelog/templates/change.md.tera

```diff
@@ -0,0 +1,45 @@
+{# This macro shows ONE change #}
+{%- macro change(c, cml="[C]", dot="[P]", sub="[S]") -%}
+
+{%- if c.meta.C and c.meta.C.value >= 7 -%}
+{%- set prio = " ‼️ HIGH" -%}
+{%- elif c.meta.C and c.meta.C.value >= 5 -%}
+{%- set prio = " ❗️ Medium" -%}
+{%- elif c.meta.C and c.meta.C.value >= 3 -%}
+{%- set prio = " Low" -%}
+{%- else -%}
+{%- set prio = "" -%}
+{%- endif -%}
+
+{%- set audit = "" -%}
+{#
+{%- if c.meta.D and c.meta.D.value == 1 -%}
+{%- set audit = "✅ audtited " -%}
+{%- elif c.meta.D and c.meta.D.value == 2 -%}
+{%- set audit = "✅ trivial " -%}
+{%- elif c.meta.D and c.meta.D.value == 3 -%}
+{%- set audit = "✅ trivial " -%}
```

Comment:
should these be the same

Reply:
The audit section is actually commented out with `{# #}` (see [here](https://github.com/paritytech/cumulus/pull/761/files#diff-85b67c64089aa634ef7aba5ee3070226e39d317eafdad142908c1c71c94d49cbR15-R27)) so the audit part is not actuve yet.

Reply:
ok